.sample.env

```diff
@@ -0,0 +1,11 @@
+COUPON_CREATOR_ADDR=
+ETHERSCAN_API_KEY=
+BINANCESCAN_API_KEY=
```

Comment:
This is missing from the list in the `README`. Should it be included with a description?

Reply:
Oh, we don't use that anymore. I will remove it.